### PR TITLE
add kujira mainnet

### PR DESCRIPTION
**Pull request by Galadrin**
mainnet on July 1st

### CODE REVIEW

**Comment by gh0stdotexe** [comment]
@Galadrin thanks for submitting this! Hopefully it's merged ASAP - we have some nodes + seeds to add!

**Comment by Galadrin** [comment]
> @Galadrin thanks for submitting this! Hopefully it's merged ASAP - we have some nodes + seeds to add!

feel free to PR on top of my patch :)
or DM the diff to apply

File: kujira/chain.json [hunk 1]
```diff
@@ -0,0 +1,62 @@
+{
+    "$schema": "../../chain.schema.json",
+    "chain_name": "kujira",
+    "chain_id": "kaiyo-1",
+    "pretty_name": "Kujira",
+    "status": "live",
+    "network_type": "mainnet",
+    "bech32_prefix": "kujira",
```
Comment (JeremyParish69) [inline kujira/chain.json:8]: Can we please register this prefix to SLIP 173?
https://github.com/satoshilabs/slips/blob/master/slip-0173.md

Comment (dylanschultzie) [reply kujira/chain.json:8]: Done: https://github.com/satoshilabs/slips/pull/1356

**Pull request merged**